cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature

```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
+
+    @sdk @IntegratedPlayer
+    Scenario: Positive Scenario : Create and validate broadcast player instance
```

Comment:
we would need to write the tc in xumo and mf config module.

Reply:
ported to config module